.github/workflows/tests.yml

```diff
@@ -95,3 +95,21 @@ jobs:
         # TODO: Delete the line below and uncomment the line beneath that
         run: prove -lmrsv webdriver/t/login.t
         #run: prove -lmrsv webdriver/t
+  
```

Comment:
I know this would normally be in another pull, but the tests have been added to this commit, so I didn't see the point in modifying the file separately when it wasn't needed.